components/my-account/Lists/ViewLists/ViewLists.spec.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+
+import { composeStories } from '@storybook/testing-react'
+import { render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import mediaQuery from 'css-mediaquery'
+
+import * as stories from './ViewLists.stories'
+import { renderWithQueryClient } from '@/__test__/utils'
+const { Common } = composeStories(stories)
+
+const createMatchMedia = (width: number) => (query: string) => ({
+  matches: mediaQuery.match(query, { width }),
+  addListener: () => jest.fn(),
+  removeListener: () => jest.fn(),
+  media: query,
+  onchange: null,
+  addEventListener: jest.fn(),
+  removeEventListener: jest.fn(),
+  dispatchEvent: jest.fn(),
+})
+
+const setup = () => {
+  const user = userEvent.setup()
+  renderWithQueryClient(<Common />)
+  return { user }
+}
+
+describe('[componenet] - ViewLists', () => {
+  it('should render loader for ViewLists', () => {
+    setup()
+    const loader = screen.getByRole('progressbar')
+    expect(loader).toBeVisible
+  })
+  it('should render ViewLists checkbox and pagination', async () => {
+    window.matchMedia = createMatchMedia(1000)
+    setup()
+    const currentUserFilterCheckbox = await screen.findByTestId('currentUserFilterCheckbox')
+    const pagination = await screen.findByTestId('pagination')
+    expect(currentUserFilterCheckbox).toBeVisible()
+    expect(pagination).toBeVisible()
+  })
+})
```

Comment:
Mock all the children component ex. ListTableMock

Naming conventions
 should render ViewLists checkbox and pagination
 should be
 should render component

"should render component", should cover
whether ListTableMock rendered or not

Missing Testcases
should display filtered wish list when user selects "Show only lists created by me" checkbox
should display next page when user clicks on next page number
should display previous page when user clicks on previous page number

Reply:
done